rclcpp_action/test/test_client.cpp

```diff
@@ -265,11 +278,37 @@ class TestClient : public ::testing::Test
   typename rclcpp::Publisher<ActionStatusMessage>::SharedPtr status_publisher;
 };
 
+class TestClientAgainstServer : public TestClient
+{
+protected:
+  void SetUp() override
+  {
+    SetUpServer();
+    TestClient::SetUp();
+  }
+
+  void TearDown() override
+  {
+    TearDownServer();
+    TestClient::TearDown();
```

Comment:
Nit: shouldn't those be closed like this?
```suggestion
    TestClient::TearDown();
    TearDownServer();
```

Reply:
Sure. At this point, it's the same.